### DISCUSSION TITLE
Few glitches I've encountered ...

### DISCUSSION THREAD

**Discussion by Dcomposer07** [apohl79/audiogridder#466] [Help]
New Audiogridder user here, love the idea and potential of this software! Game changer! 
I am experiencing a few issues so far. I'm testing this in Pro-Tools 2021.3, MacOS 10.15.7, running a local server on my iMac Pro. 

Issue #1: Attempting to load Orchestral Tools' SINE Player (either as an AU or VST) triggers a crazy infinite crash/reload loop that can only be quit by force quitting through Mac's Activity Monitor. 
(It would be amazing to be able to load the SINE Player plug-in via Audiogridder, it's the only Instrument plug-in that keeps me using Vienna Ensemble Pro which I would love to abandon entirely for various reasons)

Issue #2: When instantiating a plug-in instance of Audiogridder (instrument) in Pro-Tools, it does not automatically route the Instrument I/O to the Audiogridder plug-in like all other instrument plug-ins do Pro-Tools by automatically route themselves when instantiated. It takes a bit of time to manual route each instance when you're working on 200+ track templates, it would be so cool to have it automatically route. 

Issue #3: Attempting to duplicate an Instrument Track with an instance of Audiogridder creates an infinite feedback loop/ glitch of showing the loading the instrument on the duplicate track but never completing the task. The same ForceQuit via Mac's Activity Monitor is required to quit out of this Audiogridder glitch. 

That's all so far! I hope there's hope for these issues! Would be an unstoppable solution for us Pro-Tools composers who use many VIs and love the single instance approach and have been trying to get away from VEP. 

Thanks!
Danny

**Comment by apohl79** [comment]
> Issue #1: Attempting to load Orchestral Tools' SINE Player (either as an AU or VST) triggers a crazy infinite crash/reload loop that can only be quit by force quitting through Mac's Activity Monitor.
> (It would be amazing to be able to load the SINE Player plug-in via Audiogridder, it's the only Instrument plug-in that keeps me using Vienna Ensemble Pro which I would love to abandon entirely for various reasons)

I will try SINE player.

> Issue #2: When instantiating a plug-in instance of Audiogridder (instrument) in Pro-Tools, it does not automatically route the Instrument I/O to the Audiogridder plug-in like all other instrument plug-ins do Pro-Tools by automatically route themselves when instantiated. It takes a bit of time to manual route each instance when you're working on 200+ track templates, it would be so cool to have it automatically route.

I will have a look at how Pro Tools works here and will try to figure out what has to be done. I created feature ticket https://github.com/apohl79/audiogridder/issues/467 for this.

> Issue #3: Attempting to duplicate an Instrument Track with an instance of Audiogridder creates an infinite feedback loop/ glitch of showing the loading the instrument on the duplicate track but never completing the task. The same ForceQuit via Mac's Activity Monitor is required to quit out of this Audiogridder glitch.

I created bug ticket https://github.com/apohl79/audiogridder/issues/468 for this. I will try to reproduce this, but if you can follow https://audiogridder.com/bug-reports/ and attach the resulting files to the bug ticket, that would be helpful too.

Thanks,
Andreas

**Comment by Bholtland** [comment]
+1 for SINE player!

**Comment by Bholtland** [comment]
SINE player seems to work well now!
